### ISSUE TITLE
won't start on debian 9

### ISSUE THREAD

**Issue by modgethanc** [beakerbrowser/beaker#1344]
Operation System: Debian 9
Beaker version: 0.8.2

I managed to successfully install from the appimage, but I get the following error if I try to start Beaker:

```
 Uncaught exception: TypeError: Cannot read property 'watchWindow' of null
     at createShellWindow (/tmp/.mount_beakerj4hLye/app/resources/app.asar/background-process.build.js:1513:18)
     at ensureOneWindowExists (/tmp/.mount_beakerj4hLye/app/resources/app.asar/background-process.build.js:1596:5)
     at App.electron.app.on (/tmp/.mount_beakerj4hLye/app/resources/app.asar/background-process.build.js:3497:5)
     at App.emit (events.js:182:13)
 beaker Uncaught exception: TypeError: Cannot read property 'watchWindow' of null
     at createShellWindow (/tmp/.mount_beakerj4hLye/app/resources/app.asar/background-process.build.js:1513:18)
     at ensureOneWindowExists (/tmp/.mount_beakerj4hLye/app/resources/app.asar/background-process.build.js:1596:5)
     at App.electron.app.on (/tmp/.mount_beakerj4hLye/app/resources/app.asar/background-process.build.js:3497:5)
     at App.emit (events.js:182:13)
```

I also get a popup dialog that says: 
> Failed to open URI "beaker://shell-window/".
> The specified location is not supported.

**Comment by Butterfly** [comment]
Your pop up window error caught my eye. I JUST had that happen a couple weeks ago for something non-Beaker. Was able to re-trigger it just now by asking Thunar package manager to open `file:///asdfadfadf.pdf` (via a terminal command).

In both cases, they're likely balking at the first part there: `beaker://` for yours and `file:///` *for sure* with mine.

As soon as I removed the unnecessary `file://` part of mine, it worked fabulously the other day. That part was something that had come along with when I copied a local link from my browser's address bar/field.

Well, here's a weird. It's not throwing that error now when I try to run it one more time before posting this comment. 

THEN... I tried it with a real file, and it worked. My command line entry asking Thunar to load that `file:///` triggered it to open the file via my favored default, the Atril file viewer.

Did you try to run yours twice already? Who knows.. In the meantime, it would still need addressed somehow if that `beaker://shell-window` works everywhere else except in that instance. :smile:
